### DISCUSSION TITLE
Training on non-DRACH/other modifications

### DISCUSSION THREAD

**Discussion by hengjwj** [GoekeLab/m6anet#106] [General] [answered]
Hi Christopher, 

Is there a step-by-step (e.g. which scripts to modify) documentation on how to train m6anet on non-DRACH or other modifications? Also, how to include more features and use larger k-mers (NUM_NEIGHBORING_FEATURES in constants.py?) ? The closest I have found is #105 in Discussions but would appreciate it if you could share more details. Thanks!

Joel

P.S. I appreciate how easy it was to install and run m6anet!

**Comment by chrishendra93** [comment, marked as answer]
hi @hengjwj ,

Currently there is not a step-by-step guide on how to do that yet. The one in #105 is probably the most general advice I can provide right now but let me know if you need a more specific help on this - I might add extra documentation on this and maybe simplify the process so that one can adapt this to different modifications more readily

Christopher Hendra
